src/styles/blocks/card.scss

```diff
@@ -0,0 +1,103 @@
+* {
+  margin: 0;
+}
+
+.card {
+  display: flex;
+  color: #060b35;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 198.4px;
+  height: 406px;
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+  flex-direction: column;
+  transition-duration: $hover-duration;
+  gap: 40px;
+
+  &:hover {
+    transform: scale(1.2);
+    color: #34568b;
+  }
+
+  &__top {
+    padding-top: 32px;
+    display: flex;
+    justify-content: center;
+  }
+
+  &__product-info {
+    padding: 0 16px 16px;
+  }
+
+  &__product-name {
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+  }
+
+  &__picture {
+    width: 160px;
+    height: 134px;
+  }
+
+  &__product-code {
+    margin-top: 4px;
+    color: $secondary-text-color;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+  }
+
+  &__reviev {
+    margin-top: 16px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__price {
+    margin-top: 24px;
+    display: flex;
+    justify-content: space-between;
+
+    &-a {
+      color: $secondary-text-color;
+      font-size: 12px;
+      font-weight: 400;
+      line-height: 18px;
+    }
+
+    &-b {
+      color: $main-text-color;
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px;
+    }
+  }
+
+  &__button-link {
+    margin-top: 16px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 166px;
+    height: 40px;
+    border-radius: 5px;
+    background: $blue-color;
```

Comment:
Make sure that the `$blue-color` variable is defined and represents the desired color for the button background.